### PR TITLE
[mod] reverse log order

### PR DESCRIPTION
**Pull request by Psycojoker**
I kept being bothered that the log where in reverse order so I end up trying to find a way to change that.

With this PR new line appears at the bottom of the logs, not at the top. Dunno if you also think we should change that?

### CODE REVIEW

File: src/js/yunohost/helpers.js [hunk 1]
```diff
@@ -37,11 +37,11 @@
 
             // Add message
             $('#flashMessage .messages')
-                .prepend('<div class="alert alert-'+ level +'">'+
+                .append('<div class="alert alert-'+ level +'">'+
                               '<p>'+ message +'</p></div>');
 
             // Scroll to top to view new messages
-            $('#flashMessage').scrollTop(0);
+            $('#flashMessage').scrollTop(1000000000);
```
Comment (Psycojoker) [inline src/js/yunohost/helpers.js]: This is kinda hackish :(

I try to find a way to calculate that easily but html/js is weird and end up losing patience.

**Comment by alexAubin** [comment]
Uuuuuh it kind of works, but when I hover the mouse on the messages - I correctly see the logs in reverse (well "normal" I guess) order - but when I unhover the mouse, the message displayed in the bar is the oldest instead of the latest ... (not sure that explanation is 100% clear @.@)

**Comment by Psycojoker** [comment]
Ah, indeed >_>

Do we really care about that? I've fiddle a bit with it and couldn't find any obvious way to fix it :/

**Comment by Psycojoker** [comment]
Meh.

I should have fixed it if I've identified all those horrible corner cases.

This was really painful to write.

I've written documentation in case someone is stupid enough to work on that after me.

**Comment by alexAubin** [comment]
Meh, I ran some tests but to me it's still not intuitive what happens ... Basically I tried to : 

- Open the admin
- Go to the Update section to trigger some messages
- I see the usual messages about the update stuff going on : 
```
(info) Fetching available upgrades for system packages…
(info) Fetching available upgrades for applications…
(success) The application list yunohost has been fetched
```
- So after it's done, I see "The application list yunohost has been fetched" in the top bar
- I hover the top bar, not scrolling anything
- I hover out
- But now the message displayed is "Fetching available upgrades for system packages…" (the oldest) instead of the "The application list yunohost has been fetched" (i.e. the most recent)

Trying to hack the code a bit, I found that : 
- for some reason the `flashMessage.scrollTop()` instruction does not work ... I imagine this is related to the fact that when it's ran, the animation is still running so the bar is still not "folded" ? (Not sure tho) I was able to work around it using a `setTimeout` with a delay equal to 1 ...
- most of the time (well at least when not scrolling up or anything) the code falls into an uncovered "else" case, where I would naively expect to just add a `scrollTop(1000000)` to go back to the very bottom (but I'm also not sure to understand the computation happening for the other cases, in diagonal it sounds related to the case where the user scrolled up ?)

**Comment by alexAubin** [comment]
Closing because the log / history got entirely reworked in the vuejs rework
